rclrs/src/lifecycle_node/state.rs

```diff
@@ -107,11 +110,13 @@ impl State {
         if state_handle.is_null() {
             return Ok(());
         }
-
-        // SAFETY: By this point, we should have confirmed that state_handle still exists
-        unsafe {
-            rcl_lifecycle_state_fini(state_handle.as_mut().unwrap(), &self.allocator).ok()?;
-        }
+        
+        //if !self.from_raw {
+            // SAFETY: By this point, we should have confirmed that state_handle still exists
+            unsafe {
+                rcl_lifecycle_state_fini(state_handle.as_mut().unwrap(), &self.allocator).ok()?;
+            }
+        //}
```

Comment:
```suggestion
        // SAFETY: By this point, we should have confirmed that state_handle still exists
        unsafe {
            rcl_lifecycle_state_fini(state_handle.as_mut().unwrap(), &self.allocator).ok()?;
        }
```

Reply:
Actually, it is just the same. I should have disabled the check for any test. I have just pushed a commit with this fixed :)